proto/staking/v1beta1/staking.proto

```diff
@@ -111,10 +111,15 @@ message Validator {
       [(gogoproto.nullable) = false, (gogoproto.stdtime) = true, (gogoproto.moretags) = "yaml:\"unbonding_time\""];
   // commission defines the commission parameters.
   Commission commission = 10 [(gogoproto.nullable) = false];
-  // min_self_delegation is the validator's self declared minimum self delegation.
-  string min_self_delegation = 11 [
-    (gogoproto.moretags)   = "yaml:\"min_self_delegation\"",
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
```

Comment:
Min self delegation is redundant with exempt delegation.

Otherwise we would have to make the min self delegation automatically exempt. This would be unnecessary complexity.

Reply:
I know that we don't really use min self delegation, but I kind of like the idea for validators to have some minimum stake in the game.

Reply:
Exempt delegations are the Skin in the game. The idea of putting atoms in the validator's operator key as skin in the game does not represent how validators actually work as businesses and how custody is actually managed.

Exempt delegation models this much better. The user who exempts delegation is giving up a massive amount of flexibility with ATOMs for the purpose of growing the validators delegators. This is much more representative of the multi person partnership that most validators represent rather than putting substantial numbers of ATOMs in shared custody.

Reply:
That makes sense. Thanks for the clarification